packages/google_identity_services_web/lib/src/js_interop/package_web_tweaks.dart

```diff
@@ -0,0 +1,35 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// Provides some useful tweaks to `package:web`.
+library package_web_tweaks;
```

Comment:
CC @sigurdm re our helpers

Reply:
Thanks @sigmundch for the `has` suggestion! I'm going to try that, rather than having my own nullable getter!

However I'm going to file an issue because I think that `trustedTypes` should be `nullable` on our API. It's only available [in Chromium browsers](https://caniuse.com/mdn-api_trustedtypes), and undefined/null elsewhere.